packages/types/src/user-idp.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod"
+export const UserIDPSchema = z.object({
+  email: z.string(),
+  gender: z.string(),
+  familyName: z.string(),
+  givenName: z.string(),
+  subject: z.string(),
+})
```

Comment:
I think renaming this to OidcUser represents what the data is a lot better

https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse